app/controllers/search_controller.rb

```diff
@@ -1,59 +1,36 @@
 class SearchController < ApplicationController
-	before_filter :if_not_signed_in
+  before_filter :if_not_signed_in
```

Comment:
No, I saw about the last method inside each controller, they are repeating the code. I can declare in just one place, then reuse it!